frontend/src/lib-components/molecules/modals/InfoModal.tsx

```diff
@@ -165,19 +165,18 @@ function InfoModal({
               </ModalTitle>
               {children}
               {resolve && (
-                <ModalButtons>
+                <ModalButtons $singleButton={!reject}>
```

Comment:
what's that `$`?

Reply:
It's a [transient props](https://styled-components.com/docs/api#transient-props) that does not show up in the dom.